documents/employment-contract-no.md

```diff
@@ -0,0 +1,128 @@
+# Ansettelseskontrakt
+
+## Arbeidsområde
+
+Arbeidets art er beskrevet i den ansattes rollebeskrivelse. Arbeidet skal utøves
+i samsvar med gjeldende formål og den til enhver tid vedtatt strategi,
+innenfor rammen av de instrukser og anvisninger Bjerk AS' ledelse gir. Bjerk gir
+og forventer en betydlig grad av selvstendighet og initiativ innenfor rammene
+av rollebeskrivelsen.
+
+Arbeidstakeren kan ikke uten skriftlig godkjennelse ha annet lønnet arbeid eller
+verv.
+
+## Plikter
+
+Under sitt arbeid skal arbeidstakeren sørge for at den virksomhet arbeidstakeren
+har ansvaret for og/eller selv utøver, drives i henhold til lover, forskrifter
+og etter god forretningsskikk. Arbeidstakeren skal særlig iaktta det forhold at
+Bjerk må opptre upartisk og uavhengig i markedet. Arbeidstaker må i alt sitt
+arbeid og i all sin fremferd sørge for at det ikke oppstår grunnlag for tvil om
+Bjerk AS' uavhengige og objektive stilling og dets absolutte pålitelighet.
+
+For øvrig plikter arbeidstakeren å følge gjeldende sikkerhetsprosedyrer og
+personalhåndbøker.
```

Comment:
```suggestion
## Plikter

Arbeidstaker plikter å gjennomføre pålagt arbeid innenfor gjeldende lover,
offentlige vedtak og gyldige avtaler.

For øvrig plikter arbeidstakeren å følge gjeldende sikkerhetsprosedyrer og
personalhåndbøker.
```

Jeg fikk tilbakemelding fra @AnneMatilde at hun opplevde arbeidskontrakten veldig «advokatsk», og hun utbroderte med at ordlyden noen steder fikk henne til å føle at vi forsøkte å «fange» folk.

Derfor skrev jeg om denne noe lange og tunge setningen til hva det handler om i bunn og grunn. Noen andre forslag?

Reply:
Vedrørende «sikkerhetsprosedyrer»: Jeg har en visjon om at vi skal utforme sikkerhetsprosedyrer om ting som tofaktorautentisering, hvordan man lagrer ting her og der og hvordan vi generelt forholder oss til sikkerhet.

Som utviklere er vi eksponert både for forretningshemmeligheter og kildekode, samt intellektuell verdier så for vår egen og kundenes del bør det være en nedfestet – sånn at det er tydelig at vi tar det på alvor.

Reply:
Skjønner godt hva Anne Matilde mener. Ser bra ut det her!